src/core/1/guide/guides/kuzzle-depth/scalability/index.md

```diff
@@ -0,0 +1,230 @@
+* The cluster plugin registers a few [pipes](/core/1/plugins/plugins/pipes/), and some of them may exceed the default pipe timeouts. 
+
+Add the following to your kuzzlerc configuration file (see our [Kuzzle configuration guide](/core/1/guide/guides/essentials/configuration/)):
+
+```js
+"plugins": {
+  "common": {
+    "pipeWarnTime": 5000,
+    "pipeTimeout": 10000
+  },
+  "cluster": {
+    "privileged": true
+  }
+}
+```
+
+Once the plugin installed and configured, you can start as many Kuzzle instances as you need, and they will automatically synchronize and work together.
+
+---
+
+## Extended API
+
+The cluster plugin adds an [API controller](/core/1/plugins/plugins/controllers) named `cluster`, with the following actions defined:
+
+### health
+
+The `cluster:health` API action returns the cluster health status.
+
+#### HTTP
+
+```
+GET http://<host>:<port>/_plugin/cluster/health
+```
+
+#### Other Protocols
+
+```js
+{
+  "controller": "cluster/cluster",
+  "action": "health"
+}
+```
+
+#### Result
+
+```js
+{
+  "status": 200,
+  "error": null,
+  "controller": "cluster/cluster",
+  "action": "health",
+  "result": "ok"
+}
+````
+
+### reset
+
+The `cluster:reset` API action resets the cluster state and forces a resync.
+
+#### HTTP
+
+```
+POST http://<host>:<port>/_plugin/cluster/reset
+```
+
+#### Other Protocols
+
+```js
+{
+  "controller": "cluster/cluster",
+  "action": "reset"
+}
+```
+
+#### Result
+
+```js
+{
+  "status": 200,
+  "error": null,
+  "controller": "cluster/cluster",
+  "action": "reset",
+  "result": "ok"
+}
+````
+
+### status
+
+The `cluster:status` API action returns the current cluster status.
+
+#### HTTP
+
+```
+GET http://<host>:<port>/_plugin/cluster/status
+```
+
+#### Other Protocols
+
+```js
+{
+  "controller": "cluster/cluster",
+  "action": "status"
+}
+```
+
+#### Result
+
+```js
+{
+  "status": 200,
+  "error": null,
+  "controller": "cluster/cluster",
+  "action": "status",
+  "result": {
+    "count": 3,
+    "current": {
+      "pub": "tcp://<kuzzle node IP>:7511",
+      "router": "tcp://<kuzzle node IP>:7510",
+      "ready": true
+    },
+    "pool": [
+      {
+        "pub": "tcp://<kuzzle node IP>:7511",
+        "router": "tcp://<kuzzle node IP>:7510",
+        "ready": true
+      },
+      {
+        "pub": "tcp://<kuzzle node IP>:7511",
+        "router": "tcp://<kuzzle node IP>:7510",
+        "ready": true
+      }
+    ]
+  }
+}
+```
+
+---
+
+## How a Kuzzle cluster works
+
+### Auto-discovery and Synchronization
+
+Kuzzle nodes are synchronized by maintaining their state in the Redis server instance.  
+What this means is that, as long as Kuzzle nodes connect to the same Redis instance, they see each others and they work together.
+
+Check our [Kuzzle configuration guide](/core/1/guide/guides/essentials/configuration/) to know how to make Kuzzle connect to specific Redis instances.
+
```

Comment:
You didn't speak about zeromq for inter-node communication ?

Reply:
Hm... good point. But, should I?

Reply:
I re-read the `How a Kuzzle cluster works` section, and I actually agree: I'll add a sentence or two about 0mq :+1: